src/linagora.esn.profile/app/show/details/profile-show-details.controller.js

```diff
@@ -5,12 +5,13 @@ require('../../common/profile-api.service.js');
 angular.module('linagora.esn.profile')
   .controller('profileShowDetailsController', profileShowDetailsController);
 
-function profileShowDetailsController(profileHelpersService) {
+function profileShowDetailsController(profileHelpersService, $scope) {
   var self = this;
 
   self.$onInit = $onInit;
 
   function $onInit() {
+    $scope.user = self.user;
```

Comment:
Mh... why do we need this? Also you'll want to add some unit tests to make sure this doesn't break again.

Reply:
By the way why is it "Partially fix" :trollface:

Reply:
![image](https://user-images.githubusercontent.com/32195667/92380331-3733f700-f133-11ea-863e-bbe6de4e523f.png)
The reason for the bug is the missing $scope.user ( which is `undefined` in the picture above )

Reply:
:trollface: Oops, fixed the partially part

Reply:
Write the unit tests then :trollface:

Reply:
Question: If there is a `self.user`, why don't use `ctrl.user` in template but use $scope?

Reply:
`$scope.user` is used multiple places in`avatar.js`, and this is where the bug coming from. Using `$scope.user` is the way it used to be done before the bug appeared also, whoever changed the code forgot to pass it in.